data/421/170/397/421170397.geojson

```diff
@@ -346,7 +364,9 @@
         "wk:page":"Heredia, Costa Rica"
     },
     "wof:concordances_alt":{
-        "qs_pg:id":1200232
+        "gn:id":3623486,
```

Comment:
These should be dictionaries of lists per https://github.com/whosonfirst/whosonfirst-properties/blob/main/properties/wof/concordances_alt.json

So the existing record had it wrong, too. Ooops.

Instead it should be `"gn:id":[3623486],` & etc.

Reply:
Huh, good catch - I'll add logic for this one, too..